### PR TITLE
nit: update already found typos from #1111

### PR DESCRIPTION
**Pull request by leongross**

### CODE REVIEW

**Review by korran — APPROVED** [review]

**Review by jhand2 — APPROVED** [review]

**Review by jlmahowa-amd — APPROVED** [review]

**Comment by leongross** [comment]
In order for this to be merged, I need an approving review from a code owner @bluegate010 @FerralCoder @rusty1968 @mhatrevi @ajisaxena @vsonims @JohnTraverAmd

**Review by mhatrevi — APPROVED** [review]

**Comment by leongross** [comment]
@FerralCoder, @bluegate010, and/or @rusty1968 are required.

**Comment by leongross** [comment]
@FerralCoder @bluegate010 @rusty1968

**Review by FerralCoder — APPROVED** [review]

**Pull request merged**
